db/migrations/mysql/000088_remaining_migrations.down.sql

```diff
@@ -0,0 +1 @@
+-- Skipping it
```

Comment:
Might we expand in file on why we chose to skip? I sense a support engineer arriving here and needing additional context in the future.